### DISCUSSION TITLE
Next release 77.3-dev

### DISCUSSION THREAD

**Discussion by andrzejkokoszka** [mRemoteNG/mRemoteNG#2505] [Q&A] [answered]
Hi,
Is there a date when the next release will be published? I am interested in this feature: https://github.com/mRemoteNG/mRemoteNG/pull/2188 
Will this go in the next stable release?

**Comment by Kvarkas** [comment, marked as answer]
working on mandatory changes at the moment once have a free minute, will not say exact day as I have some haters there :) but should be soon NB release
